docs/reference.rst

```diff
@@ -21,8 +21,11 @@ Model & Components
    Source
    TimeSeriesSource
    StretchSource
+   MLCS2k2Source
    SALT2Source
    SALT3Source
+   SNEMOSource
+   SUGARSource
```

Comment:
Can you also add these to the inheritance diagram on L145?

Reply:
I updated the inheritance diagram to automatically document all classes in the models, magsystems, and bandpasses modules. That way we don't have to remember to add them in the future.